digiwf-address-service-integration-service/src/main/resources/application-local.yml

```diff
@@ -0,0 +1,55 @@
+io:
+  muenchendigital:
+    digiwf:
+      address:
+        service:
+          url: ''
+
+spring:
+  cloud:
+    function:
+      definition: functionRouter;sendMessage;sendCorrelateMessage;
+    stream:
+      bindings:
+        functionRouter-in-0:
+          destination: <topic>
+          group: <groupId>
+        sendCorrelateMessage-out-0:
+          destination: <topic>
+        sendMessage-out-0:
+          destination: <topic>
+      function:
+        routing:
+          enabled: true
+      kafka:
+        binder:
+          brokers: <brokerUrl>
+          configuration:
+            security:
+              protocol: SSL
+          consumerProperties:
+            auto:
+              offset:
+                reset: latest
+            key:
+              deserializer: org.springframework.kafka.support.serializer.JsonDeserializer
+            value:
+              deserializer: org.springframework.kafka.support.serializer.JsonDeserializer
+          producerProperties:
+            key:
+              serializer: org.springframework.kafka.support.serializer.JsonSerializer
+            value:
+              serializer: org.springframework.kafka.support.serializer.JsonSerializer
+  kafka:
+    consumer:
+      properties:
+        spring:
+          json:
+            trusted:
+              packages: '*'
+    ssl:
+      key-password: ''
+      key-store-location: ''
+      key-store-password: ''
+      trust-store-location: ''
+      trust-store-password: ''
```

Comment:
You have to activate the `local` profile to enable spring cloud stream. Is that behaviour is intended?

Reply:
This are the properties for a local run of the application.

If the application is run an a server or within a container the properties can be set in other ways (e.g. with a openshift secret). Additionally the propeties file is a kind of documentation which properties has to be set to run the application.